it/src/main/resources/tests/joins/fullOuterEquiJoin.test

```diff
@@ -3,7 +3,6 @@
 
     "backends": {
         "couchbase": "skip",
-        "mimir": "pending",
```

Comment:
We probably should add it.  By coincidence, mimir happens to get the field order in the same order as qscript is expecting.